tutor-multi-chart/values.yaml

```diff
@@ -61,3 +61,19 @@ elasticsearch:
       xpack.security.transport.ssl.certificate: /usr/share/elasticsearch/config/certs/tls.crt
       xpack.security.transport.ssl.certificate_authorities: /usr/share/elasticsearch/config/certs/ca.crt
       xpack.security.transport.ssl.verification_mode: certificate
+
+# Configuration for the metrics server chart
+metricsserver:
```

Comment:
Would it make sense to nest this block with other related blocks?

Reply:
Which other related blocks are you referring to exactly?